sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 ## Release History
 
-## 4.17.0-beta.1 (Unreleased)
+### 4.17.0 (2021-07-08)
+#### New Features
+* Adjust MicroBatchSize dynamically based on throttling rate in BulkExecutor.
 
+#### Key Bug Fixes
+* Fixed an issue where authentication context was missing in Barrier related requests.
```

Comment:
this fix some specific to AAD only.

```suggestion
* Fixed an issue where AAD authentication context was missing in Barrier related requests.
```

Reply:
Updated, thanks @moderakh

Reply:
this fix some specific to AAD only also as this is targeting end users who don't know what barrier request is maybe something like this:

```suggestion
* Fixed an issue with AAD authentication in Strong and BoundedStatelness direct mode.
```